### ISSUE TITLE
[details/maps]: Replace pie-charts

### ISSUE THREAD

**Issue by pinpox** [megaclan3000/megaclan3000#84]
![image](https://user-images.githubusercontent.com/1719781/85125303-a46a7700-b22b-11ea-9102-d331b52dcf8c.png)

- Replace the charts library with the one used in the other charts.
- Port win/loose chart
- Port used weapons chart
- Fill in table with variables
- Remove placeholder picture when done

**Comment by pinpox** [comment]
The charts library used throughout the project is [echarts](https://echarts.apache.org/). Both charts in the screenshot above are still using a different library and have to be replaced by new ones in `/templates/chart-maps.html`. Maybe the two charts can be merged into one or displayed in a different form. 

The [echarts exmaples page](https://echarts.apache.org/examples/en/index.html) shows most possiblities, the implementation should be very similar to `/templates/chart-accuracy.html`

So basically, pick a good visualisation from the echarts exmples and look at `/templates/chart-accuracy.html` to implement `/templates/chart-maps.html` mostly from scratch
